### ISSUE TITLE
Unable to install with pip or pipx

### ISSUE THREAD

**Issue by kiedtl** [pawamoy/shell-history#28]
pip output:
```
% pip install shellhistory
Collecting shellhistory
  Could not find a version that satisfies the requirement shellhistory (from versions: )
No matching distribution found for shellhistory
```

**Comment by pawamoy** [comment]
I have this problem of using dashes (or not) arbitrarily. It's `pip install shell-history` (like the repo slug, with a dash!).

**Comment by pawamoy** [comment]
Well, I really don't understand what's going on. It should be registered as `shellhistory` on PyPI: https://pypi.org/project/shellhistory/#files

Yet I'm unable to install it on my current device, but able to on another.

Will investigate thoroughly tomorrow.

**Comment by pawamoy** [comment]
Hahahaha. It was quite simple actually. As stated in the README, you need Python 3.6 or higher. I guess the error message from pip could be more helpful.

**Comment by kiedtl** [comment]
Well well well this is very interesting.

Here's my output of `python -V`:
```
Python 3.7.3
```
And lemme try again with `sudo python -m pip install shellhistory`:
```
Collecting shellhistory
  Could not find a version that satisfies the requirement shellhistory (from versions: )
No matching distribution found for shellhistory
```
*What?*

**Comment by pawamoy** [comment]
What about `sudo python -V`?

**Comment by pawamoy** [comment]
Well no you're right, it's not available for Python 3.7. It must be a mistake in pyproject.toml.

**Comment by pawamoy** [comment]
Yep: `python = "~3.6"`

It should be `^3.6`, my bad! Sorry it took so long to figure out ^^' I thought I already had fixed that (coming from a template of mine, had the same issue in another project).

Thank you for helping!

**Comment by pawamoy** [comment]
Fixed in 07f305d. Just tried to install it in a fresh Python 3.7 virtualenv, it worked 🙂

**Comment by kiedtl** [comment]
Great, it's working now! Thanks!
